Samples/nuget.config

```diff
@@ -10,6 +10,7 @@
     <packageSources>
     <clear />
         <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+        <add key="Local" value="C:\Users\erlangl\Downloads" />
```

Comment:
I see this is still a draft, so maybe I'm getting ahead of myself, but we might want to remove this :)

Reply:
We will, as soon as we have WindowsAppSDK v1.1 nugets available on a feed. 

This sample is in it's very beginning and still has a long way to go!

Thanks for looking...

Reply:
Ah I see, thank you for this sample!